src/components/common/YourCompany/YourCompany.vue

```diff
@@ -580,5 +580,8 @@ export default class YourCompany extends Mixins(
   .invalid-section .col-sm-3:first-child label {
     color: $app-red !important;
   }
+  #contact-info-edit-btn {
+    padding-right: 4px;
+  }
```

Comment:
For a tiny tweak like this, I'd use `pr-1` in the template. Might make it easier to do future tweaking.

Reply:
For sure, not sure why i did this here!

Reply:
O right, because this is a Shared Component. There is no local template to apply the helper class too.

Reply:
You're right. I didn't notice this is under a v-deep. Resolving.